README.md

```diff
@@ -79,6 +79,10 @@ This repository house companies who are known to hire directly from Africa and s
 
 39. <h4><a href="https://energi.world/join-our-team/">Energi</a></h4><p>Energi (NRG) is a next-generation Proof-of-Stake (PoS) cryptocurrency that combines smart contracts, decentralized governance, and a self-funding treasury. </p><p><strong>Location- </strong>British Virgin Island | <small>Remote</small> </p>
 
+40.  <h4><a href="https://deimos.io/">Deimos</a></h4><p> Cloud Native Developer and Security Operations Company. </p><p><strong>Location- </strong>South Africa-Cape Town | <small>Remote(South Africa, Kenya, Nigeria, Ghana)</small> </p>
```

Comment:
We need the link to its carreer page. @ddodoo

Reply:
@saviour123  I have updated the URL link.

Reply:
Recent commit has the changes